cmake/FairCMakeModulesConfig.cmake.in

```diff
@@ -8,8 +8,10 @@
 
 @PACKAGE_INIT@
 
-set_and_check(@PROJECT_NAME@_PREFIX @PACKAGE_CMAKE_INSTALL_PREFIX@)
+set_and_check(@PROJECT_NAME@_PREFIX "@PACKAGE_CMAKE_INSTALL_PREFIX@")
+set_and_check(@PROJECT_NAME@_DATADIR "@PACKAGE_PROJECT_INSTALL_DATADIR@")
+set_and_check(@PROJECT_NAME@_MODULESDIR "@PACKAGE_PROJECT_INSTALL_MODULESDIR@")
```

Comment:
Just as a note: I wanted to avoid setting too many variables, that probably nobody needs anyway.

Reply:
agreed, removed them.